package.json

```diff
@@ -40,6 +40,7 @@
   "dependencies": {
     "@adminjs/express": "^5.1.0",
     "@adminjs/typeorm": "^4.0.0",
+    "@ambire/signature-validator": "^1.4.1",
```

Comment:
This is a very light-weight dependency. We can either import it as a separate library or have the code in our own codebase and maintain it ourselves.